### PR TITLE
feat: move api and web url to vscode configuration 

### PR DESCRIPTION
**Pull request by meniRoy**
- Removed env0 api and web url from env vars
- Added env0 api and web url to extension configuration
- Added mock for unit tests
- Changed default URL to production URL

### CODE REVIEW

**Comment by meniRoy** [comment]
[add env0 URL to extension configuration](https://trello.com/c/LR0dCfB6/10803-add-env0-url-to-extension-configuration)

File: package.json [hunk 1]
```diff
@@ -19,6 +19,21 @@
   ],
   "main": "./dist/extension.js",
   "contributes": {
+    "configuration": {
+      "title": "env0",
+      "properties": {
+        "env0.apiUrl": {
+          "type": "string",
+          "required": true,
```
Comment (liranfarage89) [inline package.json:27]: `required` and `default` together? shouldn't it be one of them?
I think `required` is redundant here no?

**Review by liranfarage89 — COMMENTED** [review]
@meniRoy does the property work? are you able to hit "dev" backend using it?

**Comment by meniRoy** [comment]
> @meniRoy does the property work? are you able to hit "dev" backend using it?

yes

File: src/common.ts [hunk 2]
```diff
@@ -1,3 +1,7 @@
-export const ENV0_API_URL = process.env.ENV0_API_URL || "api.env0.com";
-export const ENV0_WEB_URL = process.env.ENV0_WEB_URL || "app.env0.com";
+import * as vscode from "vscode";
+
+export const ENV0_API_URL =
+  vscode.workspace.getConfiguration("env0").get("apiUrl") || "api.env0.com";
+export const ENV0_WEB_URL =
+  vscode.workspace.getConfiguration("env0").get("webUrl") || "app.env0.com";
```
[the remark this answers is not in the document]
Comment (meniRoy) [reply src/common.ts:6]: Even with the required and the default value, it is possible to delete the values of the configuration and leave them empty. That's the reason I added the default value here as well.
Let me know if you want me to delete this default value, here

File: package.json (hunk 1 above)
Comment (meniRoy) [reply package.json]: 54080f8bf8af1a9c1c1e8607d3ba90c164fb37a6

**Review by liranfarage89 — APPROVED** [review]

**Pull request merged**